diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index c737c93..0461627 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -270,16 +270,7 @@ module.exports = {
     return require('./Libraries/StyleSheet/PlatformColorValueTypes')
       .PlatformColor;
   },
-  get PushNotificationIOS() {
-    warnOnce(
-      'pushNotificationIOS-moved',
-      'PushNotificationIOS has been extracted from react-native core and will be removed in a future release. ' +
-        "It can now be installed and imported from '@react-native-community/push-notification-ios' instead of 'react-native'. " +
-        'See https://github.com/react-native-push-notification/ios',
-    );
-    return require('./Libraries/PushNotificationIOS/PushNotificationIOS')
-      .default;
-  },
+  
   get processColor() {
     return require('./Libraries/StyleSheet/processColor').default;
   },
